### ISSUE TITLE
Chediak-Higashi-Syndrome

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6064]
**Preferred term label:**
Chediak-Higashi-Syndrome

**Synonyms**
Chediak-Steinbrick-Higashi Syndrome, Chediak-Higashi disease, CHS

**Definition (free text, please give PubMed ID)**
A rare autosomal recessive disorder. Patients show hypopigmentation, recurrent infections, mild coagulation defects and varying neurologic problems
PMID: 18043242

**Parent term (use hpo.jax.org/app)**
Immunodeficiency

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 214500

**Gene association**
LYST (OMIM: 606897)

**Comment by pnrobinson** [comment]
The HPO does not have terms for diseases, only for the phenotypic features of diseases. The diseases are then annotated by HPO terms, see https://hpo.jax.org/app/browse/disease/OMIM:214500
